### PR TITLE
✨  Nhj consensusmerge docker

### PR DESCRIPTION
**Pull request by NHJohnson**
## Description

This is a Dockerfile and CircleCI config for the consensus_merge script for the somatic consensus workflow.  

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

- [X] Dockerfile builds on EC2 instance
- [X] Script runs with `--help` in container

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings
- [ ] I have committed any related changes to the PR

### CODE REVIEW

File: consensus-merge/1.0.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:20.04
+LABEL maintainer="Nathan Johnson <johnsonn13@chop.edu>"
+LABEL description="This is a script to create a single merged VCF from multiple single-caller somatic VCFs"
+
+RUN apt update -y && apt install -y curl
+
+# Download and install conda into /usr/bin
+ENV MINICONDA_VERSION=py38_4.9.2
+RUN curl -sL https://repo.anaconda.com/miniconda/Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh -o /tmp/miniconda.sh && \
```
Comment (migbro) [inline consensus-merge/1.0.0/Dockerfile:9]: Please consolidate, as much as possible (in this case, you can do all) the `RUN` commands as a single one, and string with `&&`. Doing so cuts the image size, in this case in half, as each `RUN` call adds a layer to the docker image and sometimes significantly inflate the size.

**Review by migbro — CHANGES_REQUESTED** [review]
This may sound crazy, but please be even more aggressive with consolidation. All of the ENVs can be clustered together, and then merge `RUN apt update -y && apt install -y curl` and 

```
RUN curl -sL https://repo.anaconda.com/miniconda/Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh -o /tmp/miniconda.sh && \
    sh /tmp/miniconda.sh -bfp /usr
```
into the other `RUN` cmd. It'll lop off like another 200MB.

**Review by dmiller15 — APPROVED** [review]

**Review by migbro — APPROVED** [review]
Awesome! In case you were curious, build size after each iteration:
1. 1.55 GB
2. 954MB
3. 705MB

**Comment by NHJohnson** [comment]
I was wondering, thanks!

**Pull request merged**